scripts/pull-review.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#
+# This script runs pull-review on a pull request
+# to intelligently assign reviewers to it.
+#
+PULL_REVIEW_GITHUB_TOKEN=$1
+PULL_REQUEST_URL=$2
+
+docker pull ghcr.io/imsky/pull-review
```

Comment:
I am not sure this line is needed, I think `docker run` will pull the image regardless.

Reply:
Of course, you are right. Removed it!